### PR TITLE
"Update mutations" schema on version 6

### PR DESCRIPTION
**Pull request by angrykoala**
# Description

This PR updates the schema in v6 with update mutations.

Translation is not yet implemented, so this is a partial feature

### CODE REVIEW

File: packages/graphql/src/api-v6/schema-generation/schema-types/TopLevelEntitySchemaTypes.ts [hunk 1]
```diff
@@ -324,6 +359,19 @@ export class TopLevelEntitySchemaTypes {
             };
         });
     }
+
+    public get updateInfo(): ObjectTypeComposer {
+        return this.schemaBuilder.getOrCreateObjectType(this.entityTypeNames.createInfo, () => {
```
Comment (MacondoExpress) [inline packages/graphql/src/api-v6/schema-generation/schema-types/TopLevelEntitySchemaTypes.ts:364]: This should be `updateInfo`
```suggestion
        return this.schemaBuilder.getOrCreateObjectType(this.entityTypeNames.updateInfo, () => {
```
But actually, I noticed that is not related to any ConcreteEntity, therefore, can be moved to the static types same for the `createInfo`

File: packages/graphql/tests/api-v6/schema/types/scalars.test.ts [hunk 2]
```diff
@@ -288,6 +306,30 @@ describe("Scalars", () => {
...
             }
 
+            input NodeTypeUpdateInput {
+              node: NodeTypeUpdateNode!
+            }
+
+            input NodeTypeUpdateNode {
+              bigInt: BigIntUpdate
+              bigIntNullable: BigIntUpdate
+              boolean: IntUpdate
```
Comment (MacondoExpress) [inline packages/graphql/tests/api-v6/schema/types/scalars.test.ts:316]: This should be BooleanUpdate

File: packages/graphql/tests/api-v6/schema/types/scalars.test.ts [hunk 3]
```diff
@@ -288,6 +306,30 @@ describe("Scalars", () => {
               stringNullable: SortDirection
             }
 
+            input NodeTypeUpdateInput {
+              node: NodeTypeUpdateNode!
+            }
+
+            input NodeTypeUpdateNode {
+              bigInt: BigIntUpdate
```
Comment (MacondoExpress) [inline packages/graphql/tests/api-v6/schema/types/scalars.test.ts:314]: I think this input field should be required

File: packages/graphql/tests/api-v6/schema/types/array.test.ts [hunk 4]
```diff
@@ -362,6 +400,42 @@ describe("Scalars", () => {
               edges: NodeTypeRelatedNodeEdgeWhere
             }
 
+            input NodeTypeUpdateInput {
+              node: NodeTypeUpdateNode!
+            }
+
+            input NodeTypeUpdateNode {
+              bigIntList: BigIntUpdate
```
Comment (MacondoExpress) [inline packages/graphql/tests/api-v6/schema/types/array.test.ts:408]: This is wrong, I expect something like: `BigIntListUpdate`

File: packages/graphql/src/api-v6/schema-model/graphql-type-names/TopLevelEntityTypeNames.ts [hunk 5]
```diff
@@ -88,4 +88,25 @@ export class TopLevelEntityTypeNames extends EntityTypeNames {
...
+
+    public get updateInput(): string {
+        return `${upperFirst(this.entityName)}UpdateInput`;
+    }
+
+    public get updateResponse(): string {
+        return `${upperFirst(this.entityName)}UpdateResponse`;
+    }
+
+    public get updateInfo(): string {
```
Comment (MacondoExpress) [inline packages/graphql/src/api-v6/schema-model/graphql-type-names/TopLevelEntityTypeNames.ts:109]: As said in another comment I would change the typename and move this as well as `CreateInfo` to some static types not related to any entity

**Review by MacondoExpress — COMMENTED** [review]
Left some comments!

File: packages/graphql/src/api-v6/resolvers/translate-update-resolver.ts [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    return async function resolve(
+        _root: any,
+        args: any,
+        context: Neo4jGraphQLTranslationContext,
+        info: GraphQLResolveInfo
+    ) {
+        const resolveTree = getNeo4jResolveTree(info, { args });
+        context.resolveTree = resolveTree;
+        const graphQLTreeCreate = parseResolveInfoTreeCreate({ resolveTree: context.resolveTree, entity });
+        const { cypher, params } = translateCreateResolver({
```
Comment (mjfwebb) [inline packages/graphql/src/api-v6/resolvers/translate-update-resolver.ts:38]: Will this be replaced with a `translateUpdateResolver` in future?
